LocalSettings.php

```diff
@@ -3591,6 +3586,15 @@
 			),
 		),
 		'+pruebawiki' => array(
+			'sysop' => array(
+				'confirmed',
```

Comment:
These rights (confirmed, rollbacker and autopatrolled) are already in "default" so there is no need to re-add them.